docs/instance_groups.md

```diff
@@ -596,3 +140,36 @@ spec:
 ```
 
 which would end up in a drop-in file on nodes of the instance group in question.
+
+## mixedInstancePolicy (AWS Only)
```

Comment:
Perhaps an example yaml snippet that contains each of the fields mentioned below?

Reply:
I can add that. Mind if I do it in a later PR?

Reply:
yea that's fine with me